messages/convert.source-behavior.md

```diff
@@ -0,0 +1,70 @@
+# summary
+
+Enable a behavior of your project source files, and then update your Salesforce DX project to implement the behavior.
+
+# description
+
+Specifically, this command updates the "sourceBehaviorOption" option in the "sfdx-project.json" file and then converts the associated local source files in your project as needed.
+
+For example, run this command with the "--behavior decomposePermissionSet" flag to start decomposing permission sets when you deploy or retrieve them. Decomposing means breaking up the monolithic metadata API format XML file that corresponds to a metadata component into smaller XML files and directories based on its subtypes. Permission sets are not decomposed by default; you must opt-in to start decomposing them by using this command. When the command finishes, your "sfdx-project.json" file is updated to always decompose permission sets, and the existing permission set files in your local package directories are converted into the new decomposed format. You run this command only once for a given behavior change.
+
+For more information about the possible values for the --behavior flag, see the "sourceBehaviorOptions" section in the https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_ws_config.htm topic.
+
+# flags.behavior.summary
+
+Behavior to enable; the values correspond to the possible values of the "sourceBehaviorOption" option in the "sfdx-project.json" file.
+
+# examples
+
+- Update your Salesforce DX project to decompose custom labels:
+
+  <%= config.bin %> <%= command.id %> --behavior decomposeCustomLabels
+
+- Display what the command would do, but don't change any existing files:
+
+  <%= config.bin %> <%= command.id %> --behavior decomposeCustomLabels --dry-run
+
+- Keep the temporary directory that contains the interim metadata API formatted files:
+
+  <%= config.bin %> <%= command.id %> --behavior decomposeCustomLabels --dry-run --preserve-temp-dir
```

Comment:
```suggestion
  <%= config.bin %> <%= command.id %> --behavior decomposeCustomLabelsBeta --dry-run --preserve-temp-dir
```

Reply:
Julit wants to keep it with its future name so we don't have to fix the examples later

Reply:
Rethinking this, because even when this feature goes GA, the `Beta` labels will still work but show a warning, right?  So it might be better to include the `Beta` label in the examples, which will always work (well, for a while at least).  Hopefully we'll remember to update the examples for GA.

Reply:
> because even when this feature goes GA, the Beta labels will still work but show a warning, right?

Not exactly. As is, there is nothinh "mapping" the `Beta` version to the future GA version. It is building the options that are allowed for the flag by reading the preset json file names:

https://github.com/salesforcecli/plugin-deploy-retrieve/pull/1015/files#diff-97a0d6c1b4d5a6a6fbce9b93c06132036ee87a73f73f04de5a59cabeb870d4a7R31

So once [these presets files](https://github.com/forcedotcom/source-deploy-retrieve/tree/main/src/registry/presets) are renamed to not include `Beta`, trying to run `--behavior decomposePermissionSetBeta` they would fail, but would suggest the new preset names (`decomposePermissionSet`)

We _could_ duplicate the preset files (one with `Beta`, one without) but I think the warning will be good enough.

Reply:
@mshanemc , Let's keep `Beta` in the example.  (sorry for back and forth).   We need to remove the "this command is beta" message anyway, so we'll update the examples at that time.  Thanks!